src/lib/components/PageSection.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  export let title: string;
+  export let testId: string | undefined = undefined;
+</script>
+
+<div class="container" data-tid={testId}>
+  <div class="title-wrapper">
+    <h3 class="title">{title}</h3>
+    <slot name="value" />
```

Comment:
Probably the slot `value` could have a more generic name. Or the component name could be more specific 🤔

Reply:
Yes, I renamed it to `end` following the convention that @peterpeterparker mentioned

Reply:
Thanks 👍